H/hsakmt_roct/build_tarballs.jl

```diff
@@ -29,7 +33,8 @@ make install
 # platforms are passed in on the command line
 # ROCT-Thunk-Interface only supports Linux, seemingly only 64bit
 platforms = [
-    Linux(:x86_64, libc=:glibc),
+    Platform("x86_64", "linux"; libc=:glibc),
+    Platform("x86_64", "linux"; libc=:musl),
```

Comment:
And I managed to get them wrong
```suggestion
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "linux"; libc="musl"),
```